src/components/Transaction/index.tsx

```diff
@@ -58,18 +118,21 @@ const TransactionComponent = ({
         </div>
         <span className="transaction__separate" />
         <div className="transaction__input__output">
-          <div className="transaction__input">
-            {transaction.display_inputs &&
-              transaction.display_inputs.map((cell: InputOutput) => {
-                return cell && <TransactionCell cell={cell} address={address} key={cell.id} />
-              })}
-          </div>
+          <CellContainerComponent cells={transaction.display_inputs} address={address} pageSize={10} />
           <img src={InputOutputIcon} alt="input and output" />
           <div className="transaction__output">
-            {transaction.display_outputs &&
-              transaction.display_outputs.map((cell: InputOutput) => {
-                return cell && <TransactionCell cell={cell} address={address} key={cell.id} />
-              })}
+            <CellContainerComponent cells={transaction.display_outputs} address={address} pageSize={10} />
```

Comment:
Set `10` as a constant will be more better.

Reply:
https://github.com/nervosnetwork/ckb-explorer-frontend/pull/38/commits/029f0d4e9e12584f2d51e7fea23dd3b148218b27